src/api.js

```diff
@@ -0,0 +1,21 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+const gameId = 'DOHDqsWYkGmCuwVtm0LV';
+
+export const postScores = async (name, score) => {
+  await fetch(`${url}${gameId}/scores`, {
+    method: 'POST',
+    body: JSON.stringify({
+      user: name,
+      score,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((result) => result.json());
+};
```

Comment:
✅ The submit button should hit the POST API endpoint and post user entries 👍🏻

Reply:
Thank you Cathella for your comments.

Cheers
 #smiles